docs/zapScanning.md

```diff
@@ -0,0 +1,17 @@
+# Using zap to scan the codebase.
+
+The zap scanner has been set up to scan the codebase when manually triggered.  The results can be read in the GitHub Action run's logs under the Baseline Zap Scan and Full Zap Scan logs.
```

Comment:
I notice this sentence “The zap scanner has been set up to scan the codebase” and I wonder if that is as accurate as it could be?